conceptpower/classes.py

```diff
@@ -157,4 +158,60 @@ def get_type(self, uri):
                 data['supertype_id'] = snode.get('supertype_id')
                 data['supertype_uri'] = snode.get('supertype_uri')
         return data
-        
\ No newline at end of file
+
+    def create(self,user,password,required_concept_data,**kwargs):
+        """
+        Adds a concept to the conceptlist by using rest api
+
+        Parameters
+        -----------
+        user: str
+             userId of the person who wants to add concept
+        password: str
+                 password of the person who wants to add concept
+        required_values: dict
+                        required keys
+                        key: "word"
+                                name of the word
+                        key: "pos"
+                                part of speech of the word
+                        key: "conceptlist"
```

Comment:
Here, what I had thought is too many arguments to a method does not look good. Also anyways we need to assign values which we obtain for a concept to a dictionary as a payload and then needs to send that. I thought instead if we can make the user to form a dictionary and send to the method, that would be better. Also I added check at the start of method for all required parameters to be present.